O21.Game/Engine/Entities.fs

```diff
@@ -3,17 +3,20 @@ namespace O21.Game.Engine
 type Player = {
     TopLeft: Point
     Velocity: Vector
+    PrevDirection: HorizontalDirection
```

Comment:
Probably we need to remove computed property `Direction` and just use record's field named `Direction` instead. IIRC left direction is only possible when special bonus taken.

Reply:
Ha, I totally forgot about that.

Yeah, I'll file an issue for it, and refactor a bit here.